dts/bindings/clock/wch,ch32v00x-hse-clock.yaml

```diff
@@ -6,3 +6,10 @@ description: WCH CH32V00x HSE Clock
 compatible: "wch,ch32v00x-hse-clock"
 
 include: [fixed-clock.yaml]
+
+properties:
+  hse-bypass:
+    type: boolean
+    description: |
+      HSE crystal oscillator bypass
+      Set the property to bypass the oscillator with an external clock.
```

Comment:
Consider using a variant of the RM text:

"If set, bypasses the oscillator and feeds the clock source directly from the outside to
the XI pin, and the XO pin is suspended."